src/lib/string_sign/string_sign.ml

```diff
@@ -77,7 +77,8 @@ let%test_module "Sign_string tests" =
 
     let%test "Sign, verify with default network" =
       let s =
-        "Now is the time for all good men to come to the aid of their party"
+        "Now is the time for all good men to come to the aid of their \
```

Comment:
Nit: revert?

Reply:
it's kind of cute with the new wording, but I'll revert, nonetheless